packages/terra-button/src/Button.jsx

```diff
@@ -82,6 +82,10 @@ const propTypes = {
    * Callback ref to pass into the dom element.
    */
   refCallback: PropTypes.func,
+  /**
+   * Just like normal tabIndex, but if isDisabled then it will be forced to -1.
+   */
+  tabIndex: PropTypes.string,
```

Comment:
should this be a number or string? i've seen design systems support both.

Reply:
No, it is not valid React to pass in a number like ```-1``` as a prop like that. At least not according to my linter settings.

Reply:
🤷 
* [semantic ui](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/modules/Checkbox/Checkbox.js#L294)
* [material ui](https://github.com/mui-org/material-ui/blob/master/packages/mui-material-next/src/Button/Button.js#L639)
* [carbon design](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Tooltip/Tooltip.js#L191)
* [dash](https://github.com/plotly/dash/blob/dev/components/dash-core-components/src/components/Textarea.react.js#L195)

Reply:
TIL, thank you. I'm going to remove the top-level prop all together based on @neilpfeiffer's comment.

Reply:
TIL, thank you.

Reply:
Interesting that Semantic and Dash still allow for one of type number or string, Material and Carbon are fine with just number.